lib/strategy.js

```diff
@@ -59,7 +73,9 @@ Strategy.prototype.authenticate = function (req, options) {
       self._verify(profile, verified);
     });
     }catch(err){
-      // Error Handling
+      // This could happen for various reasons, but often means 
+      // the saml response / assertion is not encrypted
+      return self.error(err);
```

Comment:
any way to test this?

I hope we are not silencing the error later in the code flow. does it make sense to add console.error (if we are not sure if it's throwing error or not)

Reply:
We can try releasing a beta or canary version of this package and link that in for testing locally. We'll need to pair with SSO on using an unencrypted response.